src/librustc_typeck/diagnostics.rs

```diff
@@ -2617,6 +2774,28 @@ For more information see the [opt-in builtin traits RFC](https://github.com/rust
 -lang/rfcs/blob/master/text/0019-opt-in-builtin-traits.md).
 "##,
 
+E0390: r##"
+You tried to implement methods for a mutable raw pointer (*mut T). Erroneous
```

Comment:
This error is actually for implementing on _any_ primitive, sorry.

In this case, using a trait to impl a method is the better suggestion.